### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by friendlyflame** [emavgl/oinkoin#92] [General]
I just wanted to take a moment to express my deepest thanks for all the hard work, dedication, and talent you put into creating this amazing 'expense tracker app'. Plus it's privacy friendly and foss, your hard work has not gone unnoticed, and I am forever grateful for all that you do. Thank you.

**Comment by emavgl** [comment]
Hi, I am very happy you like the app - Thanks for the message :) It made my day!
